doc/developers/performance.rst

```diff
@@ -386,10 +386,17 @@ Checkout the official joblib documentation:
 - https://pythonhosted.org/joblib
 
 
-.. _warm-restarts:
+.. _warm-starts:
 
-A sample algorithmic trick: warm restarts for cross validation
+A sample algorithmic trick: warm starts for cross validation
 ==============================================================
 
-TODO: demonstrate the warm restart tricks for cross validation of linear
-regression with Coordinate Descent.
+One of the most common tasks when doing model selection is
+ref:`cross-validation <cross_validation>`. Since it is computationally
```

Comment:
As I said above, warm_start should not be used to speed up cross validation, but parameter search. I don't think the comment you are replacing should have said cross validation. (But it made an error because cv is commonly used for parameter search.)